src/command-line/index.js

```diff
@@ -27,6 +28,10 @@ program.version(Helper.getVersion(), "-v, --version")
 // Parse options from `argv` returning `argv` void of these options.
 const argvWithoutOptions = program.parseOptions(process.argv);
 
+if (program.colors) {
+	colors.enabled = true;
```

Comment:
Is there any need to add this as an option? When would we not want colours? And if there is a situation, surely the default should be colours, and the option should be `--no-colors`?

Reply:
`isTTY` being false would disable the colors.